metricbeat/module/elasticsearch/index_summary/data.go

```diff
@@ -16,68 +16,67 @@ var (
 			"docs": c.Dict("docs", s.Schema{
 				"count":   c.Int("count"),
 				"deleted": c.Int("deleted"),
-			}),
+			}, c.DictOptional),
```

Comment:
Added these optional modifiers because there are test files that don't contain these fields.

Reply:
Instead of that, added option to `Apply` to fail only on required fields.